pandaharvester/harvestermonitor/htcondor_monitor.py

```diff
@@ -161,6 +161,8 @@ def _check_one_worker(workspec, job_ads_all_dict, cancel_unknown=False, held_tim
                             workspec.submissionHost, workspec.batchID)
             tmpLog.warning(errStr)
             newStatus = None
+    # Set supplemental error message
+    workspec.set_supplemental_error(error_code=None, error_diag=errStr)
```

Comment:
Can you set a non-None value to error_code?  You may add something like "GENRARAL_ERROR" in https://github.com/PanDAWMS/panda-harvester/blob/master/pandaharvester/harvestercore/worker_errors.py and use it for all cases until a new module is created to map errors to error codes.

Reply:
done